packages/vite/src/node/plugins/asset.ts

```diff
@@ -138,7 +138,7 @@ function fileToDevUrl(id: string, { root, base }: ResolvedConfig) {
   } else {
     // outside of project root, use absolute fs path
     // (this is special handled by the serve static middleware
-    rtn = FS_PREFIX + id
+    rtn = path.posix.join(FS_PREFIX, id)
```

Comment:
This one is not really necessary since the return line will use `path.posix.join`, but used it for consistency in case that changes later on.